src/foam/u2/filter/properties/BooleanFilterView.js

```diff
@@ -104,33 +90,23 @@ foam.CLASS({
 
   methods: [
     function initE() {
-      var self = this;
       this.addClass(this.myClass())
         .start().addClass(this.myClass('container'))
-          .start({
-            class: 'foam.u2.md.CheckBox',
-            data$: this.boolT$,
-            showLabel: true,
-            label: this.BOOL_T.label
-          }).end()
+          .tag(this.TernarySwitch, {
+            ternaryState$: this.ternaryState$,
+            width: '100%',
```

Comment:
any way to pull this out into normal/regular CSS?

Reply:
Just updated for width. Height would be non-trivial to move